src/stores/room-list-v3/RoomListStoreV3.ts

```diff
@@ -56,9 +56,13 @@ export enum RoomListStoreV3Event {
     ListsLoaded = "lists_loaded",
 }
 
-// A list of rooms in a specific space
-export type SpaceRoomsState = {
+// The result object for returning rooms from the store
+export type RoomsResult = {
+    // The ID of the active space queried
     spaceId: string;
```

Comment:
One nit: can we change this `spaceKey: SpaceKey;` to be consistent with what value it holds? Right now it looks like it only holds the roomId of a space but it can also be a meta space